go-controller/pkg/ovn/egressgw.go

```diff
@@ -580,8 +580,13 @@ func (oc *DefaultNetworkController) deletePodSNAT(nodeName string, extIPs, podIP
 		klog.V(4).Infof("Node %s is not in the local zone %s", nodeName, oc.zone)
 		return nil
 	}
+	snatMatch, err := GetNetworkScopedClusterSubnetSNATMatch(oc.nbClient, oc.GetNetInfo(), nodeName, oc.isPodNetworkAdvertisedAtNode(nodeName))
+	if err != nil {
+		return fmt.Errorf("failed to get SNAT match for podIPNets %v on node %s for network %s: %v",
```

Comment:
Looks like `podIPNets` is a pointer. shouldn't this logging corrected ? `%w` ?

Reply:
done